test/e2e/page-objects/pages/homepage.ts

```diff
@@ -6,36 +6,35 @@ import HeaderNavbar from './header-navbar';
 class HomePage {
   private driver: Driver;
 
-  private sendButton: string;
+  public headerNavbar: HeaderNavbar;
 
-  private activityTab: string;
+  private readonly activityTab =
+    '[data-testid="account-overview__activity-tab"]';
 
-  private tokensTab: string;
+  private readonly balance = '[data-testid="eth-overview__primary-currency"]';
 
-  private balance: string;
+  private readonly completedTransactions = '[data-testid="activity-list-item"]';
 
-  private completedTransactions: string;
+  private readonly confirmedTransactions = {
+    text: 'Confirmed',
+    css: '.transaction-status-label--confirmed',
+  };
 
-  private confirmedTransactions: object;
+  private readonly failedTransactions = {
+    text: 'Failed',
+    css: '.transaction-status-label--failed',
+  };
 
-  private transactionAmountsInActivity: string;
+  private readonly sendButton = '[data-testid="eth-overview-send"]';
 
-  public headerNavbar: HeaderNavbar;
+  private readonly tokensTab = '[data-testid="account-overview__asset-tab"]';
+
+  private readonly transactionAmountsInActivity =
+    '[data-testid="transaction-list-item-primary-currency"]';
 
```

Comment:
Intended change to define and initialize variables in one line to speed up development.